apps/ejabberd/test/ejabberd_listener_SUITE.erl

```diff
@@ -2,9 +2,19 @@
 
 -compile([export_all]).
 
+-include_lib("eunit/include/eunit.hrl").
+
+-import(ejabberd_helper, [copy/2,
+                          data/2]).
+
+-define(DEFAULT_PORTS, [5222, 5280, 5269]).
+-define(ALT_PORTS, [5222, 5280, 5296]). %% yes it is different
+
+
 all() ->
-    [tcp_socket_is_started_with_default_backlog,
-     tcp_socket_is_started_with_options].
+    [tcp_start_stop_reload].
+%%    [tcp_socket_is_started_with_default_backlog,
```

Comment:
ooops... just to make it run faster locally, and forgot to reenable them. Fortunately you are on guard :)